site/app/config/routing.yml

```diff
@@ -40,3 +40,7 @@ librecores.user.settings.password:
 librecores_user_resend_confirmation:
     path: /user/resend-confirmation
     controller: App\Controller\UserController::resendConfirmationEmailAction
+
+mgilet_notification:
+    resource: "@MgiletNotificationBundle/Controller/"
+    prefix: /notifications
```

Comment:
This path isn't available unless you explicitly exclude it from the UserOrgNameValidator (it could clash with a username/orgname at the same path). What is this routed for? Should it to go `/user/notification`?